### PR TITLE
Issue #15340: created InputFormatted file for section 4.1.3 Empty blocks may be concise

### PR DESCRIPTION
**Pull request by Zopsss**
#15340

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyBlocksAndCatchBlocks.java [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+    //  'Empty if block.'
+    //  ''{' is not followed by whitespace.'
+    //  ''}' is not preceded with whitespace.'
+    char[] s = {'1', '2'};
+    int index = 2;
+    if (doSideEffect() == 1) {}
+    // 3 violations above:
+    //  'Empty if block.'
+    //  ''{' is not followed by whitespace.'
+    //  ''}' is not preceded with whitespace.'
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyBlocksAndCatchBlocks.java:21]: curly braces violations are false positives from our side: https://github.com/checkstyle/checkstyle/issues/15338

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyFinallyBlocks.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          } finally {
+          } // violation above 'Empty finally block.'
+        }
+      };
+
+  interface InterfaceEndingWithSemiColon2 {
+    public void fooEmpty();
+  }
+  ; // ok until #7541
+  // violation above '';' should be separated from previous line.'
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyFinallyBlocks.java:99]: not sure if this is a false positive or negative

**Review by Zopsss — COMMENTED** [review]
extra violations

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyFinallyBlocks.java (hunk 2 above)
Comment (rdiachenko) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyFinallyBlocks.java:99]: >   ; // ok until #7541
>   // violation above '';' should be separated from previous line.'

I don't understand why we have it here. The input is to cover empty `finally` blocks that is not relevant to the aforementioned lines. @Zopsss could you remove them?

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyFinallyBlocks.java]: done

**Review by rdiachenko — APPROVED** [review]
lgtm

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java [hunk 3]
```diff
@@ -35,11 +35,21 @@ public void testEmptyBlocksAndCatchBlocks() throws Exception {
...
+    }
+
     @Test
     public void testEmptyFinallyBlocks() throws Exception {
         verifyWithWholeConfig(getPath("InputEmptyFinallyBlocks.java"));
     }
 
+    @Test
+    public void testEmptyFinallyBlocksFormatted() throws Exception {
+        verifyWithWholeConfig(getPath("InputFormattedEmptyFinallyBlocks.java"));
```
Comment (romani) [inline src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java:50]: this file is identical to original Input, please remove it from exclude.

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java (hunk 3 above)
Comment (Zopsss) [reply src/it/java/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/EmptyBlocksMayBeConciseTest.java]: done

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**